app/Domain/Integrations/Integration.php

```diff
@@ -42,7 +42,7 @@ public function contacts(): array
         return $this->contacts;
     }
 
-    public function withUrls(IntegrationUrl ...$urls): self
+    public function withUrls(IntegrationUrl...$urls): self
```

Comment:
```suggestion
    public function withUrls(IntegrationUrl ...$urls): self
```

I would expect the formatting to look like this